### ISSUE TITLE
No longer working

### ISSUE THREAD

**Issue by swehba** [liamcain/obsidian-things-logbook#18]
> **Before Submitting:** Double-check that you are running the latest version of the plugin. The bug might have already been fixed 😄

### Describe the bug

The plugin is no longer syncing. It used to work just fine but is no longer syncing events from Things.

I'm sorry this bug report is a bit vague, but it is unusual behavior that has just started to appear after the plugin has been working flawlessly for many weeks.

### Steps to reproduce

1. Make sure you have a daily note in Obsidian for today's date.
2. Make sure the daily note has a section called "## Things Logbook".
3. Configure the plugin's settings within Obsidian to look for the "## Things Logbook" section.
4. Create a new task in Things. Let's call it "Test syncing".
5. Mark it as completed.
6. Go back to Obsidian, and select `Things Logbook: Sync` from the command palette.
7. You will not see any system notification messages. I used to. (BTW, I'm running on a Mac.)
8. You will not see any updates to the the "## Things Logbook" section of today's daily note.

### Expected behavior

1. See notifications from Obsidian that Things was synced.
2. See one item — "Test syncing" added below the "## Things Logbook" section in the daily note for today.

### Screenshots

- [The plugin is installed and enabled](https://d.pr/i/jxTQan+)
- [The plugin settings](https://d.pr/i/pqWPfb+)
- [A completed task in Things](https://d.pr/i/h8mWFn+)
- [The Obsidian daily note for today after syncing](https://d.pr/i/7dn1DJ+)

#### OS

macOS 11.0.1

#### Obsidian Version (e.g. v0.10.6)

Obsidian V0.12.5

### Theme (if applicable):

(n/a)

**Comment by liamcain** [comment]
Hi Steve,

Sorry to hear it stopped working unexpectedly. The Things macOS just had a bigger update recently, but that doesn't seem to have caused any issues on my end. After triggering the "Sync" command, could you check to see if there are any errors in the DevTools? 

You can check by pressing `View > Toggle Developer Tools` 
<img width="255" alt="image" src="https://user-images.githubusercontent.com/693981/123427685-f064a000-d592-11eb-932c-030c4272e078.png">

**Comment by swehba** [comment]
Yes, indeed, there were some errors:

```
VM190:608 Uncaught (in promise) TypeError: title.trimEnd is not a function
    at eval (eval at <anonymous> (app.js:1), <anonymous>:608:26)
    at Array.forEach (<anonymous>)
    at buildTasksFromSQLRecords (eval at <anonymous> (app.js:1), <anonymous>:604:22)
    at ThingsLogbookPlugin.syncLogbook (eval at <anonymous> (app.js:1), <anonymous>:782:23)
```

**Comment by liamcain** [comment]
Hi @swehba, thank you for your patience. I just released version 0.1.14 which should fix the issue you ran into.

**Comment by swehba** [comment]
Thanks @liamcain. I very much appreciate it. I can confirm it is now working. Definitely worth a cup of coffee! Cheers.
